packages/driver/src/cypress/cy.ts

```diff
@@ -726,9 +818,12 @@ export default {
 
           queue.reset()
           queue.clear()
-          timers.reset()
+          cy.resetTimer()
 
-          testConfigOverrides.restoreAndSetTestConfigOverrides(test, Cypress.config, Cypress.env)
+          queue.reset()
+          queue.clear()
+          cy.resetTimer()
```

Comment:
This doesn't look right. Shouldn't be calling each of these twice.

Reply:
Thanks. I should have handled the merge more carefully.